Chapters/4-4-Chapter26.md

```diff
@@ -0,0 +1,384 @@
+
+"Come on," she said, worried, "Stay awake."
+
+As quickly as he slipped under, he slipped back into reality.
+
+"Who are you?" he demanded, "...I-I have nothing to give – go away!"
+
+He turned and shuffled away from us with his hand still holding the back of his head. A fit of violent coughing overcame him as if he had been struggling to hold it in and he was forced to rest against a boulder to regain his breath, still holding his free hand out to ward us off.
+
+"Leave me be," he growled, almost whispering.
+
+Pomri was insistent. She approached him, trying to reassure him that she was not there for malice. But he refused her, curling up into himself. With each refusal, she broke down a little bit more until she finally knealt before him and wept. At a safe distance, he resumed his nodding and gesticulating, a hand still behind his head.
+
+I knealt near Pomri a while with my hand on her back, imagining the kind of man he might have been, who he might have loved. I didn't know Jathi, but Pomri seemed to hold him in high regard.
+
+"There is no justice to deliver," she said under her breath. "No greater good that could ever make this right."
+
+I could feel the weight of her words stirring the same fire and vitriol that she felt in her own heart. If this was the end of life on Palinan, we would not surely not go out as a flickering flame, but as frozen statues on a battlefield of our creation.
+
+Then Pomri asked me, "Linyu. Do you have any more kanosin?"
+
+I hesitated, confused for a moment.
+
+"It... it won't work"
+
+"Won't work. What do you mean?"
+
+"Kanosin... can reverse death. But memories..."
+
+She spoke in earnest, "He can make new memories. In a new life. One without pain."
+
+"Without pain..." I murmured.
+
+I dug into my satchel, ensuring that there was only one left and I produced it with the accompanying syringe. I filled the syringe while she tore off a large piece of her guard trousers, exposing more of the leg she had illuminated, and wiped it clean from ash. A brief struggle found him on his side as she bound his arms behind him and elevated his head with her satchel below his neck. He no longer fought. Perhaps he had come to accept his anticipated death. She accepted the syringe from my hand with no sign of reluctance. Perhaps she had come to understood that the virtue of a thing is in its use.
+
+"The left heart, five fingers from the center," I said, reassuringly. "Directly in."
+
+She nodded, measuring with her fingers, priming the syringe, and plunging the needle into his heart. Jathi's eyes began to dilate as a faint blue glow, obscured by skin, filled his body and all of his muscles settled into the land.
+
+"What happens now?"
+
+It was best not to tell her the truth.
+
+"Sleep. For a whole day."
+
+"And then he rises?"
+
+"And then he rises."
+
```

Comment:
> And then he rises.

Is a common phrase used in vampyric lore. I can't help laughing here, not sure if those words were intentional.